ui/src/dashboards/actions/index.js

```diff
@@ -149,7 +171,7 @@ const templates = [
 export const getDashboardsAsync = dashboardID => async dispatch => {
   try {
     const {data: {dashboards}} = await getDashboardsAJAX()
-    const stubbedDashboards = dashboards.map(d => ({...d, templates}))
```

Comment:
ok, changed it back to what the API docs say. getting conflicting information between slack and https://github.com/influxdata/chronograf/pull/1304#issuecomment-294578021 :\